src/server/data/users/123456789/index.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
do not upload test files

Reply:
added to gitignore. they are gone from git